docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+are using the [Insiders edition], you can achieve this by adding the `pin`
+attribute in the page header:
+
+!!! example "Pin a post <!-- md:sponsors -->"
+
+    Add the `pin` attribute to your first blog post:
+
+    ```hl_lines="5"
+    ---
+    date: 2023-12-31
+    updated: 2024-01-02
+    readtime: 15
+    pin: true
+    ---
+    ```
+
+    Observe how this makes the post appear on top of the index page even though
+    its publication date is prior to other posts.
+
+### Custom slugs
+
+Another useful header attribute is `slug`, which allows you to define the slug
+for your post instead of having it auto-generated by the Blog plugin from the
+first heading.
+
+!!! example "Change slug"
+
+    If, for example, you wanted the slug to be 'ny-eve'  instead of the somewhat
+    lengthy 'happy-new-years-eve', you could add the following:
+
+    ```hl_lines="6"
+    ---
+    date: 2023-12-31
+    updated: 2024-01-02
+    readtime: 15
+    pin: true
+    slug: ny-eve
+    ---
+    ```
+
+    The URL for this post should now be
+    `http://localhost:8000/blog/2023/01/31/ny-eve/`.
+
+### Related links
+
+When your blog is part of a wider site such as technical documentation, you
+will want to provide links from blog posts into your other content. One way you
+can do this is to have a related links section. The blog plugin creates one
+for you if you provide link targets in your page header:
+
+!!! example "Add a related links section"
+
+    Add the following to a blog post:
+
+    ``` hl_lines="4-6"
+    ---
+    date: 2023-12-31
+    ...
+    links:
+      - index.md
+      - tags.md
+    ---
+    ```
+
+The nice thing here is that you do not need to provide a page title. The plugin
+will deduce the link text by applying the same logic that MkDocs uses for the
+main navigation. In fact, the syntax is the same as that of the `nav` section
+in the `mkdocs.yml`, so you can override the title if you want and even define
+subsections:
+
+!!! example "Override the page titles"
+
+    Change the link section to override the page titles:
+
+    ```hl_lines="4-6"
+    ---
+    date: 2023-12-31
+    ...
+    links:
+      - Homepage: index.md
+      - Explore tags: tags.md
+      - External links:
+        - Material documentation: https://squidfunk.github.io/mkdocs-material
+    ---
+    ```
+
+As you can see, subsections are rendered in a way similar to mobile navigation
+menus.
+
+## 6. Defining authors
+
+If your blog has more than one author then you may want to identify the author
+for each blog post. The blog plugin allows you to create a file that contains
+the author information and to then reference the authors of a particular post in
+the page header.
+
+!!! example "Create author info"
+
+    Create a file `docs/blog/.authors.yml` with this content:
+
+    ```yaml
+    authors:
+      material:
+        name: Material Team
+        description: Creator
+        avatar: https://simpleicons.org/icons/materialformkdocs.svg
+      quidfunk:
```

Comment:
Typo. Also the use of `material` as an author name is kind of misleading in subsequent examples.

Reply:
Any suggestions who else we should take? JRR Tolkien? Dostoevsky?

Reply:
We could just call it `team`? My remark was targeted towards the specific word `material` being used in too many different context other than a name for a team, i.e, as a `theme` 😉